experiment_status.md

```diff
@@ -1,3 +1,17 @@
+# Status 15-NOV-2024
+
+## Aaron's CoreCLR learnings
+
+- `PCODE` manifests itself from a myriad of places.
+  - The `MethodDesc::DoPrestub()` entry point should be referenced for many of the APIs that can produce `PCODE`. (for example, `GetStubForInteropMethod()`).
+- The ILStubResolver deletes the associated IL post JIT - `ILStubResolver::ClearCompileTimeState`.
+  - This means that when an IL stub will be the interpreted, clean-up of generated IL needs to be avoided.
+  - See `CEEInfo::getMethodInfo()` for how IL stubs are impacted, see `MethodDesc::IsWrapperStub()`.
+- Created a new `Precode` type to represent the interpreter entry point.
+  - See `MethodDescCallSite` for the common ways CoreCLR calls from C++ into managed code. Specifically, `CallTargetWorker()` is of note.
+  - The `Precode` approach seems to be a valid way to loop into the execution engine without too much disruption.
+- Passing around stub context to P/Invokes needs to be considered in the interpreter case.
```

Comment:
Alternatively, we can disable P/Invoke stub sharing with the interpreter to keep things simpler. (We have not bothered with P/Invoke stub sharing in R2R images for the same reason.)